app/services/calendar_json.rb

```diff
@@ -22,8 +22,29 @@ def self.rent(rent)
     end
   end
 
+  def self.cafe(shift, user)
+    {
+      id: shift.id,
+      title: shift.to_s,
+      start: shift.start.iso8601,
+      end: shift.stop.iso8601,
+      url: Rails.application.routes.url_helpers.cafe_shift_path(shift),
+      color: 'black',
+      backgroundColor: cafe_color(shift, user),
+      textColor: 'black'
+    }
+  end
+
   private
 
+  def self.cafe_color(shift, owner)
```

Comment:
`private` (on line 38) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.